src/core_plugins/kibana/public/doc/index.html

```diff
@@ -1,47 +1,75 @@
 <div ng-controller="doc" class="app-container">
+  <div class="kuiLocalNav">
```

Comment:
Hmm, gotcha! To give you some more context, the use of tabs in the header has been problematic for navigation in other apps (e.g. Management), and I'd like to move away from them. This hasn't been firmly decided yet within the Design team, but I think we should avoid committing to it in this UI in case we decide to completely remove it in the near future.

I think we should use a similar treatment in the Log, and allow the user to navigate to the Document and Context views using links:

* From the Document view, we can provide a link to see the Context, aligned to the right side of the Table|JSON tabs (similar to the link surfaced within the Discover table).
* From the Log view, we can provide a link to see the Single Document, aligned to the right side of the Table|JSON tabs for each document (again, similar to the link surfaced within the Discover table).

This way we have a consistent pattern in all three views (Discover, Document, and Context) for navigating to Document and Context views. What do you think?